mplc/scenario.py

```diff
@@ -752,6 +752,22 @@ def data_corruption(self):
                     f"Dirichlet distribution matrix available"
                 )
                 partner.random_labels(1.0)
+            elif self.corrupted_datasets[partner_index] == 'redundant':
+                logger.debug(
+                    f"... ... Corrupting (by adding redundancy in the data ) the whole dataset"
+                    f" of partner #{partner.id} ")
+                partner.redundant_data(1.0)
+            elif self.corrupted_datasets[partner_index] == 'duplicated':
+                copy_id = np.argmin(
```

Comment:
Why? Why not a random pick among those with more data?

Reply:
Hm yes, I was targeting a random partner amongst those with the closest dataset (in terme of data volume) 
But there is no other particular reason.